From 9e3d6f6930eb252eb8fc512617d5c8ce9057b0c4 Mon Sep 17 00:00:00 2001
From: Thomas <thomas@tuomas.co.uk>
Date: Mon, 17 Dec 2012 23:40:17 +0000
Subject: [PATCH 5/9] remove references to old messages

---
 info.json     |    2 +-
 infoscreen.py |    9 ---------
 2 files changed, 1 insertion(+), 10 deletions(-)

diff --git a/info.json b/info.json
index a4ab3af..9a66cfd 100644
--- a/info.json
+++ b/info.json
@@ -1,6 +1,6 @@
 {
 	"messages": [
-	{"color":"44 221 252", "text":"ON AIR: NONE"},
+	{"color":"44 221 252", "text":"ON AIR: Studio 2"},
 	{"color":"44 0 252", "text":"Current Show: thom"}
 	]
 }
diff --git a/infoscreen.py b/infoscreen.py
index b1ad1c5..cec3206 100644
--- a/infoscreen.py
+++ b/infoscreen.py
@@ -47,15 +47,6 @@ MINUS = 11
 
 # globals
 mode = NORMAL
-
-messages = [
-	'Infoscreen (c) Thomas Preece 2009-12'
-]
-
-messagecolors = [
-	(0, 255, 0)
-]
-
 currentmessage = 0
 
 # signal handlers
-- 
1.7.9.5

