From efdf6487f90639dee70d0a4995367f77d91d3eaf Mon Sep 17 00:00:00 2001
From: Thomas <thomas@tuomas.co.uk>
Date: Tue, 18 Dec 2012 03:09:41 +0000
Subject: [PATCH 7/9] raised clock in config

---
 config.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.py b/config.py
index 139b154..5fb036f 100644
--- a/config.py
+++ b/config.py
@@ -68,7 +68,7 @@ INFO_FNAME = 'info.json'
 SHOW_DIGITAL = True					# Should we render the digital clock?
 DIGITAL_SECONDS = False				# False - HH:MM; True - HH:MM:SS
 DIGITAL_X = CENTER_X - 215			# Define where to start drawing the clock
-DIGITAL_Y = CENTER_Y - 156			# (this is the top-left point).
+DIGITAL_Y = CENTER_Y - 250			# (this is the top-left point).
 DIGITAL_DIGIT_SPACING = 100			# How many pixels to move right after each digit
 DIGITAL_SEPARATOR_SPACING = 29		# How many pixels to move right after the separator (:)
 
@@ -82,7 +82,7 @@ DIGITAL_SEPARATOR_SPACING = 29		# How many pixels to move right after the separa
 SHOW_BACKTIMER = True				# Should we render the backtimer?
 BACKTIMER_MINUS = True				# Should we show a minus at the start?
 BACKTIMER_X = CENTER_X - 215		# Define where to start drawing the backtimer
-BACKTIMER_Y = CENTER_Y + 12			# (this is the top-left point).
+BACKTIMER_Y = CENTER_Y - 100			# (this is the top-left point).
 BACKTIMER_DIGIT_SPACING = 80		# How many pixels to move right after each digit
 BACKTIMER_MINUS_SPACING = 80		# How many pixels to move right after the minus (if it's rendered)
 BACKTIMER_SEPARATOR_SPACING = 29	# How many pixels to move right after the separator (:)
-- 
1.7.9.5

